roles/reverseproxy/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+- name: Install caddy
+  block:
+      - get_url:
+          url: https://getcaddy.com
+          dest: /tmp/installCaddy
+          mode: 0077
```

Comment:
Is there some way to get a checksum or a signature on that installer file?

Reply:
Unfortunately they don't provide checksums yet :/

Reply:
Would it be bad to simply download the install script manually for now? I feel a bit uneasy if the script is executed directly from the internet on every deployment.

Reply:
Sure, i have an idea, will take care of it :)

Reply:
there you go :)
eddb419